queries/lua/textobjects.scm

```diff
@@ -20,3 +20,23 @@
  (_) @parameter.inner)
 
 (comment) @comment.outer
+
+((function
+  . (_) . (_) . (_) @_start
```

Comment:
I think this would be more clear if you capture the by the name `(identifier)` and `(parameters)`, but not strong opinion.

Reply:
Fixed this :+1: